lib/elixir/lib/string.ex

```diff
@@ -1478,6 +1481,9 @@ defmodule String do
       iex> String.at("elixir", -10)
       nil
 
+      iex> String.at("\u0065\u0301lixir", 0)
```

Comment:
There are multiple ways of doing so though. Like picking another alphabet or using multiple accents. I don't think using \uxxxx is helpful in terms of examples. Thanks!